### PR TITLE
Fixes TimeJobs duetime too long

### PR DESCRIPTION
**Pull request by liguobao**
https://github.com/PomeloFoundation/dotNETCore-Extensions/issues/13

### CODE REVIEW

File: src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs [hunk 1]
```diff
@@ -59,17 +59,22 @@ private void StartHardTimers()
                         var invoke = y.GetCustomAttribute<InvokeAttribute>();
                         if (invoke != null && invoke.IsEnabled)
                         {
-                            int delta = 0;
+                            long delta = 0;
```
Comment (yukozh) [inline src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs:62]: ulong

File: src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs [hunk 2]
```diff
@@ -59,17 +59,22 @@ private void StartHardTimers()
...
                             if (invoke._begin == default(DateTime))
                                 invoke._begin = DateTime.Now;
                             else
-                                delta = Convert.ToInt32((invoke._begin - DateTime.Now).TotalMilliseconds);
+                                delta = Convert.ToInt64((invoke._begin - DateTime.Now).TotalMilliseconds);
```
Comment (yukozh) [inline src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs:66]: UInt64

File: src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs [hunk 3]
```diff
@@ -59,17 +59,22 @@ private void StartHardTimers()
...
                             if (delta < 0)
                             {
                                 delta = delta % invoke.Interval;
                                 if (delta < 0)
                                     delta += invoke.Interval;
                             }
+                            if(delta > Int32.MaxValue)
```
Comment (yukozh) [inline src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs]: same as before

**Review by yukozh — CHANGES_REQUESTED** [review]

File: src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs [hunk 4]
```diff
@@ -96,18 +101,22 @@ private void StartDynamicTimers()
                     JobTimers.Remove(x.Id);
                     JobStatus.Remove(x.Id);
                 }
-                int delta = Convert.ToInt32((x.Begin - DateTime.Now).TotalMilliseconds);
+                ulong delta = Convert.ToUInt64((x.Begin - DateTime.Now).TotalMilliseconds);
                 if (delta < 0)
                 {
-                    delta = delta % x.Interval;
+                    delta = delta % (uint)x.Interval;
```
Comment (yukozh) [inline src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs]: Could you use Convert.ToUInt64 instead of explict conversion?

**Review by yukozh — CHANGES_REQUESTED** [review]

File: src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs (hunk 4 above)
Comment (liguobao) [reply src/Pomelo.AspNetCore.TimedJob/TimedJobService.cs]: ok,I know.

**Review by yukozh — APPROVED** [review]

**Pull request merged**